lib/matplotlib/backends/backend_wx.py

```diff
@@ -1023,20 +1026,31 @@ def _onKeyUp(self, evt):
         evt.Skip()
         FigureCanvasBase.key_release_event(self, key, guiEvent=evt)
 
+    def _set_capture(self, capture=True):
```

Comment:
@QuLogic Sorry, I do not understand the meaning of your response --  If *what* is required? 

The call signature is part of the documentation.  Turning a keyword arg into a positional arg  makes the code less readable.  Won't change.